app/src/main/java/app/insti/fragment/BodyFragment.java

```diff
@@ -241,9 +241,14 @@ public void onClick(View view) {
                     @Override
                     public void onResponse(Call<Void> call, Response<Void> response) {
                         if (response.isSuccessful()) {
+
                             body.setBodyUserFollows(!body.getBodyUserFollows());
-                            new updateDbBody().execute(body);
```

Comment:
He changed this probably because it was overriding the change he made to the TextView and this won't be necessary if he uses the cleaner method you suggested.